cves/kernel/CVE-2016-8650.yml

```diff
@@ -335,9 +350,11 @@ stacktrace:
     Answer must be true or false.
     Write a note about how you came to the conclusions you did, regardless of
     what your answer was.
-  any_stacktraces:
-  stacktrace_with_fix:
-  note:
+  any_stacktraces: true
+  stacktrace_with_fix: true
+  note: |
```

Comment:
Capitalize and add punctuation. Remove the extra space between "vulnerability's" and "pointer".